extensions/wikia/AffiliateService/AffiliateServiceHooks.class.php

```diff
@@ -83,6 +83,9 @@ public static function onWikiaSkinTopScripts( Array &$vars, &$scripts ) {
 		// only enable it onNS 0 AND dev env
 		$vars['wgAffiliateEnabled'] = $isProperNamespace && $isProperCommunity;
 
+		// add translations from AffiliateService.i18n.php
+		$vars['disclaimerMessage'] = wfMessage('affiliate-service-disclaimer')->escaped();
```

Comment:
`escaped()` is intended to escape HTML and avoid XSS. You can just use `->plain()` instead.